versions.txt

```diff
@@ -27,10 +27,10 @@ autoinstrumentation-python=0.44b0
 
 # Represents the current release of DotNet instrumentation.
 # Should match autoinstrumentation/dotnet/version.txt
-autoinstrumentation-dotnet=1.2.0
+autoinstrumentation-dotnet=1.6.0
```

Comment:
This should not be updated due to https://github.com/open-telemetry/opentelemetry-operator/issues/2542.